modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'appeals_api/form_schemas'
+require 'decision_review/utilities/saved_claim/service'
```

Comment:
all the files living in `appeals_api` belong to Lighthouse so no need to add your changes here, so that should simplify the PR a bit 😁

Reply:
Thanks, will remove those changes 👍